site/ja/tutorials/load_data/images.ipynb

```diff
@@ -0,0 +1,3254 @@
+{
```

Comment:
<p>These links should be changed from '"en/" to "ja"</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/427/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Thanks.  Fixed.</p>

Reply:
<p>I checked that this is fixed.</p>